libs/client/cornflow_client/core/instance_solution.py

```diff
@@ -335,22 +333,42 @@ def add_time_to_ts(ts: datetime, weeks=0, days=0, minutes=0, seconds=0) -> datet
         return ts + timedelta(days=7 * weeks + days, minutes=minutes, seconds=seconds)
 
     @staticmethod
-    def add_time_to_date_string(string: str, weeks=0, days=0, minutes=0, seconds=0) -> str:
+    def add_time_to_date_string(
```

Comment:
why do we have all these dates functions in the instance_solution class? shouldn't we have them all under a time package that can be itself used from the instance? or a different package/ class that can be called from outside instance? With the current way of working the instance is polluted with a huge number of date-specific methods that are not necessarily relevant for all problems.

Reply:
We have these functions in another library which we could use as a dependency but I haven't got around in doing the change.